upgrade_test.py

```diff
@@ -316,11 +317,13 @@ def _upgrade_node(self, node, upgrade_sstables=True):
             InfoEvent(message='upgrade_node - ended to "upgradesstables_if_command_available"').publish()
 
     @truncate_entries
-    @decorate_with_context(ignore_abort_requested_errors)
     # https://github.com/scylladb/scylla/issues/10447#issuecomment-1194155163
     def rollback_node(self, node, upgrade_sstables=True):
-        # pylint: disable=too-many-branches,too-many-statements
+        self._rollback_node(node=node, upgrade_sstables=upgrade_sstables)
 
+    @decorate_with_context(ignore_abort_requested_errors)
+    def _rollback_node(self, node, upgrade_sstables=True):
```

Comment:
why this is introduced ? to skip the truncate check ?

Reply:
Yes, I wrote it in the test:
```
# Call "_rollback_node" to prevent running truncate test
```